deploy_enclave.sh

```diff
@@ -52,9 +53,14 @@ pushd "terraform/projects/enclave/${ENCLAVE}/${STATE}"
     then
         aws-vault exec ${PROFILE} -- terraform apply --target module.network.aws_vpc_endpoint.ec2
     fi
-    if [ "${TERRAFORM_ACTION}" == "apply" ] 
+    if [ "${TERRAFORM_ACTION}" == "apply" ]
     then
-        aws-vault exec ${PROFILE} -- terraform plan -target $TARGET --out "${planfile}"
+        if [ -z "$TARGET" ]
```

Comment:
Indentation looks slightly off here.

Reply:
Thanks have corrected the indentation, and typo.